### PR TITLE
WINDUP-596 - Enhancements to the EJB and JPA processors.

### PR DESCRIPTION
**Pull request by bradsdavis**

### CODE REVIEW

File: rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/util/HibernateDialectDataSourceTypeResolver.java [hunk 1]
```diff
@@ -13,30 +13,45 @@
...
+        else if(StringUtils.contains(dialect, "Derby")) {
+            return "Derby";
+        }
+        else if(StringUtils.contains(dialect, "Pointbase")) {
+            return "Pointbase";
+        }
+        else if(StringUtils.contains(dialect, "Postgres")) {
+            return "Postgres";
         }
-        else if (StringUtils.contains(dialect, "H2Dialect"))
```
Comment (jsight) [inline rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/util/HibernateDialectDataSourceTypeResolver.java]: Why remove H2Dialect?

File: rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/util/HibernateDialectDataSourceTypeResolver.java [hunk 2]
```diff
@@ -13,30 +13,45 @@
...
-        {
-            return "HyperSQL";
+        else if(StringUtils.contains(dialect, "Ingres")) {
+            return "Ingres";
+        }
+        else if(StringUtils.contains(dialect, "Derby")) {
+            return "Derby";
+        }
+        else if(StringUtils.contains(dialect, "Pointbase")) {
+            return "Pointbase";
```
Comment (jsight) [inline rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/util/HibernateDialectDataSourceTypeResolver.java]: I think that this should be "PointBase".

Comment (bradsdavis) [inline rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/util/HibernateDialectDataSourceTypeResolver.java]: Probably an oversight.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/759/<h2>Failed Tests: <span class='status-failure'>2</span></h2><h3><a name='org.jboss.windup.tests:windup-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/759/org.jboss.windup.tests$windup-tests/testReport'>org.jboss.windup.tests:windup-tests</a>: <span class='status-failure'>2</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/759/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureJEEExampleTest/testRunWindupJEEExampleMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureJEEExampleTest.testRunWindupJEEExampleMode</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/759/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSourceModeTest/testRunWindupSourceMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureSourceModeTest.testRunWindupSourceMode</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/128/
Test PASSed.

**Pull request merged**